### PR TITLE
gatekeeper operator catalog deployment

### PR DESCRIPTION
**Pull request by marzianor9**

### CODE REVIEW

**Comment by SchSeba** [comment]
/hold
until validation

**Review by SchSeba — COMMENTED** [review]
@marzianor9 just a general question there is a reason you check the pods and not just the deployment obect the replica == ready ?

**Comment by marzianor9** [comment]
@SchSeba i wanted to check on the deployment, but to my understanding "readyReplicas" on deployment does not equal running pods, if it does i would be happy to change that.

**Review by SchSeba — APPROVED** [review]
/lgtm
/approve

File: feature-configs/deploy/gatekeeper/subscription.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: gatekeeper-operator-product
+  namespace: openshift-operators
```
Comment (SchSeba) [inline feature-configs/deploy/gatekeeper/subscription.yaml:5]: @mmirecki why we use this namespace?

Comment (mmirecki) [reply feature-configs/deploy/gatekeeper/subscription.yaml:5]: @SchSeba this is the default ns we install to when installing from operatorhub

**Pull request merged**